tests/serde.rs

```diff
@@ -218,7 +218,7 @@ fn numbers() {
         // Big ints that are too large or small should error
         let u64_max_bigint = std::u64::MAX.serialize(&bigint_serializer).unwrap();
         from_value::<i64>(u64_max_bigint).unwrap_err();
-        let negative_one = -1_i64.serialize(&bigint_serializer).unwrap();
+        let negative_one = -(1_i64.serialize(&bigint_serializer).unwrap());
```

Comment:
Are you sure you didn't mean the opposite - `(-1_i64).serialize(...)`? I'm not sure if negation operator is overloaded for JsValue.

Reply:
But also, if you use those values only as temporaries and not for testing serialization, you might as well just create `BigInt::from(...)` directly.

Reply:
Or, if we go there, even `BigInt::from(i128::MIN)` since you only care about value being out of bounds of `i64`/`u64` and that one is the easiest to get.

Reply:
Good point! Changed